kubernetes-aws-go/Pulumi.yaml

```diff
@@ -0,0 +1,10 @@
+name: ${PROJECT}
+description: ${DESCRIPTION}
+runtime: go
+template:
+  description: A Go program to create a Kubernetes cluster on AWS
+  important: true
```

Comment:
I'm actually not sure whether we should or should not be setting this for these arch templates. I didn't set it for the static-website templates, and it doesn't look like Luke's PR sets it, either. This used to toggle whether the template appeared in the list of options with `pulumi new` called with no arguments, but the CLI I have today doesn't seem to use this value anymore. (I [asked about this today](https://pulumi.slack.com/archives/C9SGX9QA1/p1663283005951889) and it sounds like this might be a bug, but at the same time, the UX around this comment is also changing, so we may or may not use this property going forward.) 

If it were me, I'd probably leave it out for now, just for consistency, but yeah -- we'll know more soon. 😄

Reply:
@cnunciato Are you referring to the `important: true` line?

Reply:
Yep, sorry -- I see now I could've made that clearer. 😂

Reply:
Fixed in 8f2808f. The `important: true` line is removed from `Pulumi.yaml` for all languages present in this PR.